src/register-org/components/organisation-services-access/organisation-services-access.component.ts

```diff
@@ -87,21 +77,38 @@ export class OrganisationServicesAccessComponent extends RegisterComponent imple
     this.cancelRegistrationJourney();
   }
 
-  public setFormControlValues(): void {
+  private setFormControlValues(): void {
+    this.selectedServices = this.registrationData.services;
     this.services.forEach((service) => {
-      if (this.registrationData.services.includes(service.value_en)) {
-        service.selected = true;
-      }
+      service.selected = this.selectedServices.includes(service.key);
     });
+    if (this.registrationData.otherServices) {
+      this.showOtherServicesInput = true;
+      this.selectedServices.push('NONE');
+      this.services.find((service) => service.key === 'NONE').selected = true;
```

Comment:
@johnbenjamin-hmcts - is `services.find()` always guaranteed to return a truthy result?

Reply:
`NONE` is always a part of the master hard-coded list of data `ORGANISATION_SERVICES`. So, the `find` will always return a result.